virt/release_notes/virt-4-14-release-notes.adoc

```diff
@@ -195,6 +213,10 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 //CNV-28723 Release note: PREVIEW
 
 //CNV-28944 Release note: Preview Cluster level eviction strategy change
+* You can configure any of the following xref:../../virt/nodes/virt-node-maintenance.adoc#eviction-strategies[cluster eviction strategies]:
```

Comment:
Just a nit about the bullet list -- these are actually VM eviction strategies. The "new" feature is that now the user can configure eviction strategies at the cluster level instead of at the template level for individual VMs. 

I recommend rephrasing, perhaps like this: "You can now configure a VM eviction strategy for the xref:../../virt/nodes/virt-node-maintenance.adoc#virt-configuring-cluster-eviction-strategy-cli_virt-node-maintenance[entire cluster]."

Reply:
Just noticed there is no TP notice for the feature. I'll add that in a separate PR.

Reply:
Shouldn't the Preview features be in the TP section?

Reply:
Done